.gitignore

```diff
@@ -0,0 +1 @@
+vendor
```

Comment:
Please, add new line at the end of the files. You should be able to configure your IDE to do it automatically.